src/ranged.cpp

```diff
@@ -341,29 +424,52 @@ dealt_projectile_attack Creature::projectile_attack( const projectile &proj_arg,
     return attack;
 }
 
+static double erfinv( double x )
```

Comment:
A black-box unit test would be useful. Not sure about the latter as-yet. I don't want to ask you to write something unless its clear what the benefits are. The graphs are already much smoother so this does suggest your revised version resolves much of the original concern.